tsdb/operations/default.json

```diff
@@ -1,89 +1,291 @@
-    {
-      "name": "index",
-      "operation-type": "bulk",
-      "bulk-size": {{bulk_size | default(5000)}},
-      "ingest-percentage": {{ingest_percentage | default(100)}}
+{
+  "name": "index",
+  "operation-type": "bulk",
+  "bulk-size": {{bulk_size | default(5000)}},
+  "ingest-percentage": {{ingest_percentage | default(100)}}
+},
+{
+  "name": "default",
+  "operation-type": "search",
+  "body": {
+    "query": {
+      "match_all": {}
+    }
+  }
+},
+{
+  "name": "default_1k",
+  "operation-type": "search",
+  "body": {
+    "query": {
+      "match_all": {}
     },
-    {
-      "name": "default",
-      "operation-type": "search",
-      "body": {
-        "query": {
-          "match_all": {}
+    "size": 1000
+  }
+},
+{
+  "name": "date-histo-entire-range-1m",
+  "operation-type": "search",
+  "index": "tsdb-1m",
+  "body": {
+    "size": 0,
+    "aggs": {
+      "date": {
+        "date_histogram": {
+          "field": "@timestamp",
+          "fixed_interval": "1m"
         }
       }
-    },
-    {
-      "name": "default_1k",
-      "operation-type": "search",
-      "body": {
-        "query": {
-          "match_all": {}
+    }
+  }
+},
+{
+  "name": "date-histo-entire-range-1h",
+  "operation-type": "search",
+  "index": "tsdb-1h",
+  "body": {
+    "size": 0,
+    "aggs": {
+      "date": {
+        "date_histogram": {
+          "field": "@timestamp",
+          "fixed_interval": "1h"
+        }
+      }
+    }
+  }
+},
+{
+  "name": "date-histo-entire-range-1d",
+  "operation-type": "search",
+  "index": "tsdb-1d",
+  "body": {
+    "size": 0,
+    "aggs": {
+      "date": {
+        "date_histogram": {
+          "field": "@timestamp",
+          "fixed_interval": "1d"
+        }
+      }
+    }
+  }
+},
+{
+  "name": "date-histo-entire-range",
+  "operation-type": "search",
+  "index": "tsdb",
+  "body": {
+    "size": 0,
+    "aggs": {
+      "date": {
+        "date_histogram": {
+          "field": "@timestamp",
+          "fixed_interval": "1d"
+        }
+      }
+    }
+  }
+},
+{
+  "name": "date-histo-memory-usage-hour",
+  "operation-type": "search",
+  {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
+  "index": "k8s",
+  {%- else %}
+  "index": "tsdb",
+  {% endif %}
+  "body": {
+    "size": 0,
+    "aggs": {
+      "by_date": {
+        "date_histogram": {
+          "field": "@timestamp",
+          "fixed_interval": "1h"
         },
-        "size": 1000
+        "aggs": {
+          "ts": {
+            "time_series": {
```

Comment:
I think we should also add a pipeline aggregator here. Since the next change, we're trying to do is pushing down the pipeline aggregator computation to time series aggregator on data node. With that change the response of the time_series agg would change too, it will not return time series buckets but the result of the associated pipeline aggregations.

I think a sum pipeline aggregator is ok here.